lib/main.dart

```diff
@@ -90,6 +95,11 @@ class _RandomWordsState extends State<RandomWords> {
 
   Widget _buildRow(WordPair pair) {
     final alreadySaved = _saved.contains(pair);
+    Color tileColor = _randomColor.randomColor(colorBrightness: ColorBrightness.light);
+    while (tileColor == previousColor){
```

Comment:
Are you sure this won't cause infinite loop?
Isn't there any inbuilt parameter to avoid same colors?

Reply:
At least I didn't see this possibility

Reply:
Getting a random value in a way that it is always unique (or at least isn't equal to some other value) is an entirely separate mathematical problem (people spend their lifetime resolving it), and `do {} while (x != y)` is usually the simplest way to bypass it (even though mathematically it means that in theory you can stuck in an infinite loop when all your new values repeat themselves)